### PR TITLE
Prepartions for cpan publication

### PR DESCRIPTION
**Pull request by sirtoobii**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #  Copyright (C) 2020 Tobias Bossert
 
-AC_INIT([wg-meta],m4_esyscmd([tr -d '\n' < VERSION]),[tobias.bossert@fastpath.ch])
```
Comment (oetiker) [inline configure.ac:3]: do you still need configure? if so, I would try to NOT keep the version number in 2 places.

File: MANIFEST [hunk 2]
```diff
@@ -0,0 +1,32 @@
+AUTHORS
+bin/wg-meta
+CHANGES
+COPYRIGHT
+lib/Wireguard/WGmeta.pod
```
Comment (oetiker) [inline MANIFEST]: hmmm having a pod without matching library seems odd ... I would probaly add this one directly into the wg-meta.pl file

File: MANIFEST [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+lib/Wireguard/WGmeta/Cli.pod
```
Comment (oetiker) [inline MANIFEST:5]: and this into the README.md

there is no need to have a file at every directory level

File: configure.ac (hunk 1 above)
Comment (sirtoobii) [reply configure.ac:3]: Removed automake altogether, I think `ExtUtils::MakeMaker` is sufficient here. The version number is now just in the `VERSION` file and propagates to the occurrences in the code when calling `make`.

**Review by oetiker — APPROVED** [review]
ship it :)

**Pull request merged**